### DISCUSSION TITLE
Is there a sort of "rule of thumb" to forecast storage usage over time?

### DISCUSSION THREAD

**Discussion by vitobotta** [plausible/analytics#1328] [Self-Hosted Support] [answered]
Hola :) I was wondering if it's possible to somehow get an idea of the storage that is used by Plausible especially in Clickhouse for X page views or something like that? Thanks!

**Comment by Vigasaurus** [comment, marked as answer]
I don't know about anything specific as far as forecasting goes, but for some context - my current instance is: 
![image](https://user-images.githubusercontent.com/19434157/133339274-9a6663ff-ba0d-43cc-87d6-8e6f44285692.png)

with 13.7M events (11.7M pageviews and 2M custom goals), and 840k sessions. So take from that what you will.

**Comment by vitobotta** [comment]
Thanks @Vigasaurus ! That doesn't look bad. Thanks again
